error 190919-ensure-axiom-immutable.pyv:8:96: Only immutable relations can be referenced inside an axiom
program has resolution errors.
mypyvy exiting with status 1
